pkg/gui/filetree/file_manager_test.go

```diff
@@ -1,12 +1,21 @@
 package filetree
 
 import (
+	"os"
 	"testing"
 
+	"github.com/gookit/color"
 	"github.com/jesseduffield/lazygit/pkg/commands/models"
 	"github.com/stretchr/testify/assert"
+	"github.com/xo/terminfo"
 )
 
+func TestMain(m *testing.M) {
+	color.ForceSetColorLevel(terminfo.ColorLevelNone)
```

Comment:
Why not move this line to within `TestRender`?

Reply:
You're right! I was initially expecting this test to require "teardown". But it works just fine as the test setup, without the need for ```TestMain```